src/sprout/Controllers/Admin/OperatorAdminController.php

```diff
@@ -408,6 +434,36 @@ public function _deletePreSave($item_id)
         }
     }
 
-}
 
+    /**
+     * Remove an operator login lock by deactivating recent failed attempts in db
+     * @param int $item_id Operator ID
+     * @return void Redirects
+     */
+    public function unlockOperator($item_id)
+    {
+        $item_id = (int) $item_id;
+        $redirect = 'admin/intro/operator/';
+        $username = 'All operators';
+        $conditions = [];
+        $params = [];
+
+        $conditions[] = ['date_added', '>', date('Y-m-d H:i:s', time() - AdminAuth::LOGIN_LIMIT_SECONDS)];
+
+        if ($item_id)
```

Comment:
I think I'd prefer to see `if (!empty($item_id))  {` but this is of course valid